lib/trino-plugin-toolkit/pom.xml

```diff
@@ -33,6 +33,17 @@
             <artifactId>configuration</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>http-client</artifactId>
+            <exclusions>
```

Comment:
io.airlift.node is needed only for tests. If not excluded here dependency-scope-maven-plugin will complain

Reply:
Where do we use them in tests ?

Reply:
Instead of excluding this - Can we add `node` as a rutime dependency 
```

        <!-- used by tests but also needed transitively -->
        <dependency>
            <groupId>io.airlift</groupId>
            <artifactId>node</artifactId>
            <scope>runtime</scope>
        </dependency>